### PR TITLE
Helm charts for Livy-0.5.0

### PR DESCRIPTION
**Pull request by akshayharyani**

### CODE REVIEW

File: charts/livy-0.50/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Helm Chart for Livy-0.5.0
+
+[Livy](https://livy.incubator.apache.org/) provides a REST API for Apache Spark.
```
Comment (AlexNavara) [inline charts/livy-0.50/README.md:3]: Let's avoid 'Apache' word.

File: charts/livy-0.50/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+### Install command
+`helm install livy ./livy-chart `
+
+This will create the helm chart in the `default` namespace. To create the chart in a different existing namespace use the flag
+` -n sampletenant `.
+Please note that if you are using PVC, the pvc should exist in the same namespace.
+
+To set the tenant namespace use the flag `--set tenantNameSpace=sampletenant` during installation
+
+Integrations for Spatrk History server and metastore can be customized in the Values.yaml file.
```
Comment (AlexNavara) [inline charts/livy-0.50/README.md:18]: Typo in word 'Spark'

File: charts/livy-0.50/README.md (hunk 1 above)
Comment (akshayharyani) [reply charts/livy-0.50/README.md]: I have updated this.

File: charts/livy-0.50/README.md (hunk 2 above)
Comment (akshayharyani) [reply charts/livy-0.50/README.md]: Thanks for pointing out. I have corrected this.

**Pull request closed without merging**